### PR TITLE
Systemd support for CentOS and RHEL

### PR DESCRIPTION
**Pull request by rveznaver**
First try at fixing #85.

### CODE REVIEW

**Comment by rayrod2030** [comment]
I'll test this change tonight.  Looks great thought!

**Comment by rayrod2030** [comment]
I'm having some issues getting tests to pass in vagrant and against EC2.  I'm assuming these tests are passing for you?

**Comment by rveznaver** [comment]
@rayrod2030 Long time no see. What kind of issues in EC2? I don't test on AWS, only in VirtualBox.

**Comment by avinashkolluru** [comment]
Waiting for the merge

**Comment by rayrod2030** [comment]
Hi @rveznaver would you mind merging master into this and I can try to test it again today.  Would like to get this merged soon if possible.  Sorry for the delay!

**Comment by rayrod2030** [comment]
This is great thanks @rveznaver!

File: attributes/default.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@
 # Init system to use
 default['mesos']['init']    = case node['platform']
                               when 'debian' then 'sysvinit_debian'
+                              when 'centos', 'amazon', 'scientific'
+                                node['platform_version'].to_i >= 7 ? 'systemd' : 'upstart'
```
Comment (ovaistariq) [inline attributes/default.rb]: CentOS/RHEL do not use upstart. CentOS/RHEL versions < 7 uses sysvinit system, so this would fail on CentOS versions < 7.

Comment (rveznaver) [inline attributes/default.rb]: Hi @ovaistariq, in RHEL 6 sysvinit has been replaced by upstart. Please see [RHEL 6 Technical Notes](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/6.0_Technical_Notes/deployment.html). Versions lower than 6 are not supported by this cookbook.

Comment (ovaistariq) [inline attributes/default.rb]: OK cool. I didn't see the version mentioned in metadata. What are the plans around the next release of the cookbook with this fix for CentOS/RHEL 7 support?

Comment (avinashkolluru) [inline attributes/default.rb]: +1 waiting for the release

Comment (rayrod2030) [inline attributes/default.rb]: @ovaistariq @rveznaver Looking to cut a new release today.

Comment (avinashkolluru) [inline attributes/default.rb]: @rayrod2030 Any updates on the release?

**Pull request merged**